docs/Configuring-Pantheon/Consensus-Mechanisms.md

```diff
@@ -20,24 +20,22 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
 Properties to consider when comparing Clique and IBFT 2.0 are: 
 
 * Immediate finality 
-* Liveness 
+* Validators availability
 * Speed 
-* Minimum number of validators 
+* Type of Byzantine-fault-tolerance
 
 ### Immediate Finality 
 
 IBFT 2.0 has immediate finality. When using IBFT 2.0 there are no forks and all valid blocks are included in the main chain.
 
 Clique does not have immediate finality. Implementations using Clique must be aware of forks and chain reorganizations occurring. 
 
-### Liveness 
+### Validators availability
 
-Clique is more fault tolerant than IBFT 2.0. Clique tolerates up to half to the validators failing. IBFT 2.0 networks 
-tolerate up to (n-1)/3 faulty nodes. For example, in an IBFT 2.0 network of:
-
-* 3, no bad node are tolerated
-* 4-6, 1 bad node is tolerated 
-* 7-9, 2 bad nodes are tolerated 
+IBFT 2.0 requires at least 2/3 of the validators (rounded up to the next integer) to be always online for the network to
+be able to create new blocks and progress.
+Clique only requires more then 1/2 of the validators (rounded up to the next integer) to be always online for the
```

Comment:
use the same language as in the first paragraph - Clique requires at least 1/2 of the validators ...

Reply:
Done